nemo/collections/tts/parts/preprocessing/features.py

```diff
@@ -75,13 +73,27 @@ def _get_feature_filepath(
         feature_filepath "<feature_dir>/<feature_name>/speaker1/audio1.pt"
     """
     _, audio_filepath_rel = get_audio_filepaths(manifest_entry=manifest_entry, audio_dir=audio_dir)
-    feature_filepath = feature_dir / feature_name / audio_filepath_rel.with_suffix(".pt")
+    feature_filepath = feature_dir / feature_name / audio_filepath_rel.with_suffix(".npy")
```

Comment:
could you double-check .npy or .pt? The docstring seems use .pt.

Reply:
Updated docstrings to `.npy`